test/langtools/tools/javac/8278078/ValidThisAndSuperInConstructorArgTest.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * @test /nodynamiccopyright/
```

Comment:
This is a positive test, so we typically add copyright in those.

Reply:
done